### PR TITLE
V5 changelog

### PR DESCRIPTION
**Pull request by shellygr**
Naming convention:
 - PRs for features that are in design should have the "proposal" label
 - PRs for features that haven't landed yet should have the "future" label
 - PRs for upcoming releases should have the "release" label
 - PRs with new documentation for existing features should have the "existing feature" label

Before requesting review:
 - Make sure there is a ticket in the DOC board in Jira
 - Make sure CI is passing
   - Spell check failure may require adding backticks around code or updating `spelling_wordlist.txt`
   - See `README.md` for information about style and markdown syntax
   - If the CI Details link gives a 404, you need to log in to readthedocs.com
 - Add link to generated documentation here
   - you can find this by following the read the docs link from the CI check
 - Ask for help in #documentation

Jira ticket: TODO
Link to generated documentation: TODO

### CODE REVIEW

File: docs/prover/cli/options.md [hunk 1]
```diff
@@ -820,3 +820,20 @@ splits generated here is equal to `2^n` where `n` is the initial splitting depth
 practice).
 
 
+
+(--allow_solidity_calls_in_quantifiers)=
```
Comment (mdgeorge4153) [inline docs/prover/cli/options.md:824]: This section is already in the document

File: docs/prover/changelog/prover_changelog.md [hunk 2]
```diff
@@ -5,6 +5,47 @@ Prover Release Notes
 ```{contents}
 ```
 
+5.0.5 (November 21, 2023)
+-------------------------
+### CVL
+- [feat] Allowing calling Solidity functions from within CVL hooks
+- [feat] {ref}`direct-storage-access`
+- [feat] Support for exhaustive parametric methods. Now `method f` calls will check for all methods in all contracts in Scene. The set of checked contracts can be limited with `--parametric_contracts Contract1 Contract2`
```
Comment (mdgeorge4153) [inline docs/prover/changelog/prover_changelog.md:13]: I would probably add a link to the changes guide here.

File: docs/prover/changelog/prover_changelog.md [hunk 3]
```diff
@@ -5,6 +5,47 @@ Prover Release Notes
...
+- [bugfix] Disallow declaring method variables (aka `method f`; declarations) outside the top-level scope of a rule. They could still be declared as rule and CVL function arguments
```
Comment (mdgeorge4153) [inline docs/prover/changelog/prover_changelog.md:14]: Here also

File: docs/prover/changelog/prover_changelog.md [hunk 4]
```diff
@@ -5,6 +5,47 @@ Prover Release Notes
...
+- [bugfix] Remove assume/assert notation from `DELETE` summary qualifiers
+- [bugfix] Disallow Solidity calls in CVL quantifier bodies
+- [bugfix] Support the '$' sign in identifiers (specifically for Solidity functions)
+- [UX] When non-reverting calls lead to an 'empty function' because all paths revert, show an alert in the rule report
+
+### Performance
+- [feat] New parallel splitter, can be enabled with `--prover_args '-newSplitParallel true'`
```
Comment (mdgeorge4153) [inline docs/prover/changelog/prover_changelog.md:21]: Link to the option.

**Review by mdgeorge4153 — CHANGES_REQUESTED** [review]
A few nits

File: docs/prover/changelog/prover_changelog.md (hunk 4 above)
Comment (shellygr) [reply docs/prover/changelog/prover_changelog.md:21]: I don't think we have one

File: docs/prover/cli/options.md (hunk 1 above)
Comment (shellygr) [reply docs/prover/cli/options.md]: fixed

File: docs/prover/changelog/prover_changelog.md (hunk 2 above)
Comment (shellygr) [reply docs/prover/changelog/prover_changelog.md:16]: I added a header instead

**Pull request merged**